site2/docs/schema-evolution-compatibility.md

```diff
@@ -30,19 +30,26 @@ For more information, see [Schema compatibility check strategy](#schema-compatib
 
 ### How does Pulsar support schema evolution?
 
-1. When a producer/consumer/reader connects to a broker, the broker deploys the schema compatibility checker configured by `schemaRegistryCompatibilityCheckers` to enforce schema compatibility check. 
+The process of how Pulsar supports schema evolution is described as follows.
 
-   The schema compatibility checker is one instance per schema type. 
+1. The producer/consumer/reader sends the `SchemaInfo` of its client to brokers. 
    
-   Currently, Avro and JSON have their own compatibility checkers, while all the other schema types share the default compatibility checker which disables schema evolution.
-
-2. The producer/consumer/reader sends its client `SchemaInfo` to the broker. 
+2. Brokers recognize the schema type and deploy the schema compatibility checker `schemaRegistryCompatibilityCheckers` for that schema type to enforce the schema compatibility check. By default, the value of `schemaRegistryCompatibilityCheckers` in the `conf/broker.conf` or `conf/standalone.conf` file is as follows.
    
-3. The broker knows the schema type and locates the schema compatibility checker for that type. 
+   ```properties
+   schemaRegistryCompatibilityCheckers=org.apache.pulsar.broker.service.schema.JsonSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.AvroSchemaCompatibilityCheck,org.apache.pulsar.broker.service.schema.ProtobufNativeSchemaCompatibilityCheck
+   ```
+
+   :::note
+
+   Each schema type corresponds to one instance of schema compatibility checker. Currently, Avro, JSON and Protobuf have their own compatibility checkers, while all the other schema types share the default compatibility checker which disables the schema evolution. In a word, schema evolution is only available in Avro, JSON and Protobuf schema.
```

Comment:
```suggestion
   Each schema type corresponds to one instance of schema compatibility checker. Currently, Avro, JSON, and Protobuf have their own compatibility checkers, while all the other schema types share the default compatibility checker which disables the schema evolution. In a word, schema evolution is only available in Avro, JSON, and Protobuf schema.
```

Reply:
Use commas to separate items in a series of three or more. Use a comma before the conjunction that precedes the final item. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.b82f2ay5cpsc

Reply:
Updated. Thank you.